packages/yarnpkg-parsers/sources/resolution.ts

```diff
@@ -12,6 +12,9 @@ export type Resolution = {
 };
 
 export function parseResolution(source: string): Resolution {
+  if (source.indexOf('**') !== -1)
+    throw new Error(`The resolution '${source}' includes a glob pattern which is discouraged in v2 since it has no longer an effect.`)
```

Comment:
Even in v1 the glob patterns are mostly useful and may lead to confusion.

```suggestion
    throw new Error(`The resolution '${source}' includes a glob pattern. Glob patterns have been removed since their behaviours don't match what you'd expect.`)
```